jolly-jellyfish/src/page_maker/fixtures/webpages.yaml

```diff
@@ -0,0 +1,17 @@
+- model: page_maker.webpage
+  pk: 1
+  fields:
+    name: mypage
+    date_created: 2020-08-02 16:25:04+00:00
+    author: 1
+    thumbnail: thumbnails/test1.jpg
+    template_used: 1
+    votes: 0
+    user_title: thats my page
+    user_text_1: asdafgssag
+    user_text_2: sdgagdasd
+    user_text_3: xxcvxcvbxcvbxcvbxcvbbcvbxc
+    user_image_1: images/admin/img1.jpg
+    user_image_2: images/admin/img2.jpg
+    user_image_3: images/admin/img3.jpg
+    user_image_4: images/admin/img4.jpg
```

Comment:
Are these default images that come with a normal Django install? (Just wondering... Since otherwise, let's move them under a separate directory for test files :-)

Reply:
Just a placeholder since we don't need real files.